state/factory/workingset.go

```diff
@@ -434,6 +434,8 @@ func (ws *workingSet) pickAndRunActions(
 			switch errors.Cause(err) {
 			case nil:
 				// do nothing
+			case action.ErrChainID:
```

Comment:
if we receive a tx with wrong chainID, should simply discard and continue processing?
similar to `action.ErrHitGasLimit` below